shared/profile/dumb.js

```diff
@@ -170,9 +170,9 @@ const propsBase: RenderProps = {
       height: 578,
     },
   },
-  onClickAvatar: console.log('on click avatar'),
-  onClickFollowers: console.log('on click followers'),
-  onClickFollowing: console.log('on click following'),
+  onClickAvatar: () => console.log('on click avatar'),
```

Comment:
Thanks for fixing this. Wasn't expecting to see it in this commit, though. Separate PR next time please?